commit 326046739801a380c5457ef4c87bce8fb95497ba
Author: Sergei Trofimovich <slyfox@gentoo.org>
Date:   Sun Aug 10 22:12:28 2014 +0300

    systools info: fix warning about C compiler (message said about linker)
    
    Summary: Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
    
    Test Plan: build-tested
    
    Reviewers: austin
    
    Reviewed By: austin
    
    Subscribers: phaskell, simonmar, relrod, ezyang, carter
    
    Differential Revision: https://phabricator.haskell.org/D132

diff --git a/compiler/main/SysTools.lhs b/compiler/main/SysTools.lhs
index 1c1c52c..72fa19b 100644
--- a/compiler/main/SysTools.lhs
+++ b/compiler/main/SysTools.lhs
@@ -809,10 +809,10 @@ getCompilerInfo' dflags = do
             )
             (\err -> do
                 debugTraceMsg dflags 2
-                    (text "Error (figuring out compiler information):" <+>
+                    (text "Error (figuring out C compiler information):" <+>
                      text (show err))
                 errorMsg dflags $ hang (text "Warning:") 9 $
-                  text "Couldn't figure out linker information!" $$
+                  text "Couldn't figure out C compiler information!" $$
                   text "Make sure you're using GNU gcc, or clang"
                 return UnknownCC)
   return info
